src/metatrain/experimental/nanopet/model.py

```diff
@@ -499,7 +499,7 @@ def load_checkpoint(cls, path: Union[str, Path]) -> "NanoPET":
 
         # Load the checkpoint
         checkpoint = torch.load(path, weights_only=False, map_location="cpu")
-        model_hypers = checkpoint["model_hypers"]
+        model_hypers = checkpoint["model_data"]
```

Comment:
should the variable name also be updated?
```suggestion
        model_data = checkpoint["model_data"]
```

Reply:
Thanksss